portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,7 +24,8 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns messages. */
-@WebServlet("/data")
+/*
+*@WebServlet("/data")
```

Comment:
any reason why we are commenting these lines instead of deleting them?

Reply:
No reason, it has been deleted!